main/strongswan/APKBUILD

```diff
@@ -17,10 +17,6 @@ makedepends="$depends_dev linux-headers python2 sqlite-dev openssl-dev curl-dev
 install="$pkgname.pre-install"
 subpackages="$pkgname-doc $pkgname-dbg"
 source="https://download.strongswan.org/$pkgname-$_pkgver.tar.bz2
-	0205-ike-Adhere-to-IKE_SA-limit-when-checking-out-by-conf.patch
-	1001-charon-add-optional-source-and-remote-overrides-for-.patch
-	1002-vici-send-certificates-for-ike-sa-events.patch
-	1003-vici-add-support-for-individual-sa-state-changes.patch
```

Comment:
@bugfest I merged this some time ago. Did you verify that those patches have been upstreamed?

Reply:
Hi @ncopa, I remember checking it back when I requested this update. I've double checked today and looks good / all those were already upstreamed in 5.8.0: 

For reference:
```
$ git remote -v
origin  git://git.strongswan.org/strongswan.git (fetch)
origin  git://git.strongswan.org/strongswan.git (push)
$ git checkout 5.8.0
$ for patch in ../alpinelinux-aports/main/strongswan/*.patch; do
patch -R -p0 -s -f --dry-run < $patch >&/dev/null && echo "Missing patch: $patch" || echo "Already patched: $patch";
done
Already patched: ../alpinelinux-aports/main/strongswan/0205-ike-Adhere-to-IKE_SA-limit-when-checking-out-by-conf.patch
Already patched: ../alpinelinux-aports/main/strongswan/1001-charon-add-optional-source-and-remote-overrides-for-.patch
Already patched: ../alpinelinux-aports/main/strongswan/1002-vici-send-certificates-for-ike-sa-events.patch
Already patched: ../alpinelinux-aports/main/strongswan/1003-vici-add-support-for-individual-sa-state-changes.patch
```